contrib/ast-db-manage/config/versions/39583bd1dc8a_add_match_request_uri_attribute_to_.py

```diff
@@ -0,0 +1,20 @@
+"""add match_request_uri attribute to identify
+
+Revision ID: 39583bd1dc8a
+Revises: 74dc751dfe8e
```

Comment:
We usually don't have multiple alembic changes going on at the same time but unfortunately, you got caught again.  You're going to have to rebase your PR on current master again and change the Revises and down_revision to 8fce8496f03e

Reply:
ok